account_move_line_report_xls/report/move_line_list_xls.py

```diff
@@ -134,9 +134,9 @@ def __init__(self, name, table, rml=False, parser=False, header=True,
                 'header': [1, 13, 'text', _render("_('Maturity Date')")],
                 'lines':
                 [1, 0,
-                 _render("line.date_maturity.val and 'date' or 'text'"),
+                 _render("line.date_maturity and 'date' or 'text'"),
                  _render(
-                     "line.date_maturity.val \
+                     "line.date_maturity \
```

Comment:
To put a string on multiple line you can write

    ("start of string"
     " continuing string"
     " end of string")